.goreleaser.yaml

```diff
@@ -0,0 +1,81 @@
+project_name: apm-lambda-extension
+
+before:
```

Comment:
It changes the make goals that likely are used by the dev team, can you clarify whether this new way of using `goreleaser` is good for them when running the licenser validations?

Reply:
Moved several scripts to Makefile rules

https://github.com/elastic/apm-aws-lambda/pull/362/commits/02aa3fe7c40693271994b80ee86e89a0585f180e
https://github.com/elastic/apm-aws-lambda/pull/362/commits/b5323fa5cadad6531ffa956875457419abe8f28c
https://github.com/elastic/apm-aws-lambda/pull/362/commits/6b3bb5aca734423a653af535bca4d72799af510d
https://github.com/elastic/apm-aws-lambda/pull/362/commits/e527f43daf19eeb30cfd25dbad82ee9f39fa3987
https://github.com/elastic/apm-aws-lambda/pull/362/commits/818ba189182d8b9f02416284e1d75d785068b3d9

Reply:
@kruskall can you please also check this and suggest any adjustments to the Makefile if necessary? Thank you!

Reply:
I left a few comments, some of the tasks are also used for development/testintg. 

Apologies, I think I'm missing some context here, is there an issue with the current makefile ? I see the building logic has been moved to goreleaser but is there any special reason behind it ?

Reply:
@reakaleek 

I left a few comments, some of the tasks are also used for development/testintg.

Apologies, I think I'm missing some context here, is there an issue with the current makefile ? I see the building logic has been moved to goreleaser but is there any special reason behind it ?

Reply:
The main reason is that the current scripts were created with running on ARM CI runners in mind. Hence, in Jenkins we would just run the whole script on an ARM instance in parallel to the AMD build.

However, in GH Actions, we don't have ARM runners available by default, unless we would self-host them. Which is currently not an option. (Maybe in the future).

Therefore, we decided to use GoReleaser as it easens the multi-architecture build of artifacts.